_docs/organizations-and-collections.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Organizations and Collections
+permalink: /docs/publisher-tutorials/organizations-and-collections/
+---
+
+# Organizations and Collections
+
+## Organizations
+Organizations are landing pages for collaborations, institutions, consortiums, companies, etc. that allow users to showcase tools and workflows. This is achieved through the creation of collections, which are groupings of related tools and workflows. The users of an organization do not need to own the tools or workflows in any way, the tools and workflows just have to be published. Collections can be thought of as a playlist on a Music Streaming Service where tools and workflows are analogous to individual songs. They can be shared publicly, and the user does not need to own them.
+
+### Creating an organization
+To create an organization request, go to the [organizations](https://dockstore.org/organizations) page and select `Create Organization Request`. Any user can request to create an organization by filling out the following form. For now, the request must be approved by a Dockstore curator in order to be public. Until it is, you are still able to edit it, add collections, add members, etc.
```

Comment:
It's in the previous sentence, but I still think it would read better if it said `Until it is approved,...`.